### PR TITLE
Feat/sc 80 footer

### PR DESCRIPTION
**Pull request by davidgilbertson**
Jira: https://govausites.atlassian.net/browse/SC-80

After only several weeks, the footer task is complete. Huzzah.

### CODE REVIEW

File: src/components/FooterContainer/Footer/Footer.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+            </div>
+          </div>
+        )}
+
+        <div className={style.staticLinksWrapper}>
+          {links.map(link => (
+            <a
+              key={link.href}
+              href={link.href}
+              target="_blank"
```
Comment (micapam) [inline src/components/FooterContainer/Footer/Footer.js]: Why do these links open in new tab/window?

File: src/components/FooterContainer/Footer/Footer.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+        <img
+          className={style.dtaLogo}
+          src={dtaLogo}
+          alt="dta"
+        />
+      </div>
+
+      <div className={style.wordsWrapper}>
+        {!!props.category && (
```
Comment (micapam) [inline src/components/FooterContainer/Footer/Footer.js:62]: This seems bizarre. Why are these category/subcategory links included here? :S

**Review by micapam — CHANGES_REQUESTED** [review]

File: src/components/FooterContainer/Footer/Footer.js (hunk 2 above)
Comment (micapam) [reply src/components/FooterContainer/Footer/Footer.js:62]: (This is not a change suggestion just a comment)

Comment (davidgilbertson) [reply src/components/FooterContainer/Footer/Footer.js:62]: I concur, and can't answer that. But it increased the complexity significantly so I was happy to do it.

File: src/components/FooterContainer/Footer/Footer.js (hunk 1 above)
Comment (davidgilbertson) [reply src/components/FooterContainer/Footer/Footer.js]: I thought it was defined somewhere, but I don't see it in the task. I'll remove it.

Comment (davidgilbertson) [reply src/components/FooterContainer/Footer/Footer.js]: Fixed

**Review by micapam — APPROVED** [review]

**Pull request merged**